formats/corporate_information_page.jsonnet

```diff
@@ -21,6 +21,7 @@
     "terms_of_reference",
     "petitions_and_campaigns",
     "accessible_documents_policy",
+    "modern_slavery_statement",
```

Comment:
Could this list be sorted into alphabetical order (either as another commit, or another PR), so it makes things easier to update